### PR TITLE
Nightwatch Test Uplift

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
This PR is purely to test the recently proposed toolkit changes.

@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,14 +53,14 @@
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.jsx .",
     "lint:scss": "lerna run lint:scss",
-    "pretest": "npm run lint",
+    "nightwatch": "nightwatch",
+    "pretest-disable": "npm run lint",
```
Comment (bjankord) [inline package.json]: why change this to `pretest-disable`?

File: packages/terra-alert/package.json [hunk 2]
```diff
@@ -48,12 +48,8 @@
...
-    "test": "npm run test:spec && npm run test:nightwatch-default",
-    "test:spec": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
-    "test:all": "npm run test:nightwatch-default && npm run test:nightwatch-chrome && npm run test:nightwatch-firefox && npm run test:nightwatch-safari",
-    "test:nightwatch-default": "SPECTRE_TEST_SUITE=terra-alert node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js",
-    "test:nightwatch-chrome": "SPECTRE_TEST_SUITE=terra-alert node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js chrome",
-    "test:nightwatch-firefox": "SPECTRE_TEST_SUITE=terra-alert node ../../node_modules/terra-toolkit/lib/scripts/nightwatch.js firefox",
-    "test:nightwatch-safari": "SPECTRE_TEST_SUITE=terra-alert node ../../node_modules/terra-toolkit/lib/scripts/nightwatch-non-parallel.js safari"
+    "test": "npm run test:jest && npm run test:nightwatch",
+    "test:jest": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
+    "test:nightwatch": "$(cd ..; npm bin)/nightwatch -c ../../nightwatch.conf.js"
```
Comment (bjankord) [inline packages/terra-alert/package.json]: this is cleaned up nicely! 👍

File: package.json (hunk 1 above)
Comment (emilyrohrbough) [reply package.json]: ah, yeah, I need to change back

File: packages/terra-modal/tests/nightwatch/modal-spec.js [hunk 3]
```diff
@@ -1,144 +1,135 @@
...
-  'custom props': (browser) => {
-    browser
-      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/modal-tests/custom-props`)
-      .assert.attributeEquals('div[role="document"]', 'id', 'custom-props');
-  },
-};
+// // eslint-disable-next-line import/no-extraneous-dependencies
+// const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+//
+// module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
```
Comment (bjankord) [inline packages/terra-modal/tests/nightwatch/modal-spec.js]: What's going on with this test? Are we gonna hold off on it due to stability of the test?

Comment (emilyrohrbough) [reply packages/terra-modal/tests/nightwatch/modal-spec.js]: Should be fixed in this [commit](https://github.com/cerner/terra-core/pull/751/commits/3e3f95b0c3ddb29f70eb7128aafde48d148695bb#diff-0257b38edc76b2ad44acf8f8871dc0f7R5)

**Pull request merged**